src/widgets/ticket-detail/components/ticket-summary/ticket-summary.css.ts

```diff
@@ -0,0 +1,75 @@
+import { style } from "@vanilla-extract/css";
+
+import { themeVars } from "@shared/styles/theme.css";
+
+// 전체 래퍼
+export const container = style({
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "stretch",
+  paddingTop: "2rem",
+  paddingBottom: "2rem",
+  paddingInline: "1.6rem",
```

Comment:
```suggestion
  padding: "2rem 1.6rem",
```
이렇게 나타내는거 더 나을것 같아요!